assets/src/scripts/components/hydrograph/markers.js

```diff
@@ -13,11 +13,45 @@ function circleMarker({r, x, y, domId=null, domClass=null}) {
     if (domClass !== null) {
         circle.attr('class', domClass);
     }
-    return circle;
-}
+    if (fill != null) {
```

Comment:
Why are you drawing a second circle instead of adding the fill to the first circle?

Reply:
So, there might be some SVG shape capability, that I haven't discovered yet, but while `fill` works pretty good and much as you would expect if it was a color, I couldn't figure out a way to have a pattern that has a background color.

What I ended up doing was creating shape with the color I want, and then overlayed another shape over it that has the pattern. This approach what's creates the colored mask blocks with the diagonal hashing in both the plot and legend.

Reply:
I think this is worthy of a comment in the code